src/client/components/pages/statistics.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2018 Akhilesh Kumar <akhilesh5991@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import * as bootstrap from 'react-bootstrap';
+import * as utilsHelper from '../../helpers/utils';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+const {PageHeader, Table} = bootstrap;
+const {formatDate} = utilsHelper;
+
+/**
+ * Renders the document and displays the topEditors table.
+ * @returns {ReactElement} a HTML document which displays the topEditors table
+ * in thre statistics page
+ */
+
+function TopEditorsTable(props) {
+	const {editors} = props;
+	return (
+		<div>
+			<div>
+				<h2 className="text-center">Top 10 Editors</h2>
+			</div>
+			<Table
```

Comment:
If the line would be under 80 characters, you can put all the tags on one line :)